### ISSUE TITLE
Support renaming list of servers

### ISSUE THREAD

**Issue by tonyxiao** [bobthecow/genghis#101]
Displaying database urls alone can be very hard to understand and navigate when database urls themselves are not very descriptive. It would be very useful to allow user to rename the stored databases. (even if only persisted in cookie)

**Comment by bobthecow** [comment]
I agree. I'd like to have an "advanced" database DSN form — one that has checkboxes and inputs for the things that are otherwise crammed into query params — and this would fit nicely there.
